src/crimson/osd/osd.cc

```diff
@@ -98,6 +98,7 @@ OSD::OSD(int id, uint32_t nonce,
     tick_timer{[this] {
       update_heartbeat_peers();
       update_stats();
+      osd_check_down();
```

Comment:
What's the reason behind using `tick_timer`? Wouldn't be enough to check once ,when a new `OSDMap` is received?

Reply:
I'll recheck that, maybe i didn't waited enough, but, after the client ended with osd stop, i didn't get refreshed OSDMap. but, let me re-check it

Reply:
you are right, even with the tick_timer, we won't get any advantage, we will still wait for the update OSDMap anyway..